enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/ShiroAuthSubject.java

```diff
@@ -99,7 +115,7 @@ public boolean allowsSchemaWrites()
     @Override
     public String name()
     {
-        return "AUTH";
+        return subject.getPrincipal().toString();
```

Comment:
I wonder if we should do something about these error messages in Kernel instead:
String.format( "Write operations are not allowed for `%s` transactions.", transaction.mode().name() )

Having the username in there reads a bit strange